javalib/src/main/scala/java/io/File.scala

```diff
@@ -1,9 +1,603 @@
 package java.io
 
-class File private () extends Serializable with Comparable[File] {
-  def this(path: String) = this()
-  def this(parent: String, child: String) = this()
-  def this(parent: File, child: String) = this()
+import scala.collection.mutable.UnrolledBuffer
+
+import scala.annotation.tailrec
+import scalanative.runtime.GC
+import scalanative.native._, stdlib._, stdio._, string._, Nat._, dirent._
+import scalanative.posix.unistd._
+
+class File(_path: String) extends Serializable with Comparable[File] {
```

Comment:
This one was ported from Harmony too. Please make sure all harmony-based code has the `// Ported from Apache Harmony` comment.